### DISCUSSION TITLE
How often is feeds fetched? Can I reduce the time between fetches?

### DISCUSSION THREAD

**Discussion by mizzunet** [FreshRSS/FreshRSS#6029] [Q&A] [answered]
How can I increase the number of fetches by reducing the time between each fetches?
Say, I have subscribed to a feed and I want to fetch the feed in each 30 minutes?

Thanks (:

**Comment by Frenzie** [comment, marked as answer]
In settings → Archiving → `Do not automatically refresh more often than` you can reduce the default maximum refresh rate, or you can do it per feed.
